### PR TITLE
Origin prophets abb195

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Additions into a work record (Genealogy of the Apostles), ellimination of its doublet.

### CODE REVIEW

File: 5001-6000/LIT5849GenealogyApostles.xml [hunk 1]
```diff
@@ -1,77 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
```
Comment (eu-genia) [inline 5001-6000/LIT5849GenealogyApostles.xml:1]: https://betamasaheft.eu/Guidelines/?q=delete&id=create
If it is clear that a record is superfluous, delete it, properly document this in the commit, make sure the deleted identifier is not used elsewhere, update existing references, but also **add a relation in the record which survives**, like the following.


                  `<relation name="betmas:formerlyAlsoListedAs" active="IDofthesurvivedrecord" passive="IDofthedeletedrecord"></relation>`

If you are sure that it is a doublet and delete a record make sure to update all records pointing to it (BNFet64, BAVet10, LIT2422Tarika)!

File: 5001-6000/LIT5549TribesApostles.xml [hunk 2]
```diff
@@ -65,17 +68,33 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                        <citedRange unit="page">404–405</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:Grebaut1913Apotres"/>
+                        <citedRange unit="page">207–208</citedRange>
+                    </bibl>
                 </listBibl>
                 <listRelation>
                     <relation name="saws:formsPartOf" active="LIT5549TribesApostles" passive="LIT1461Gadlah"/>
                 </listRelation>
```
Comment (eu-genia) [inline 5001-6000/LIT5549TribesApostles.xml:96]: add
 `<relation name="betmas:formerlyAlsoListedAs" active="LIT5549TribesApostles" passive="LIT5849GenealogyApostles"></relation>`
if you are sure that these are doublets and the second is removed

Comment (DenisNosnitsin1970) [reply 5001-6000/LIT5549TribesApostles.xml:102]: ok

File: 1001-2000/LIT1811Liveso.xml [hunk 3]
```diff
@@ -59,7 +66,57 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                <relation name="lawd:hasAttestation" active="LIT6156OriginProphets" passive="BNFabb195"/>
+            </listRelation>
+               </div>
+            <div type="edition">
+              <note>The division into text parts follows <ref type="mss" corresp="BNFabb195"/>.</note>
+               <div type="textpart" subtype="chapter" xml:id="Chronology">
+                  <label>On the chronology of the Prophets</label>
+               </div>
+               <div type="textpart" subtype="chapter" xml:id="Isaiah">
+                  <label>On Prophet Isaiah</label>
```
Comment (thea-m) [inline 1001-2000/LIT1811Liveso.xml:80]: ...the Prophet...?

Comment (DenisNosnitsin1970) [reply 1001-2000/LIT1811Liveso.xml:80]: It is plural.

Comment (thea-m) [reply 1001-2000/LIT1811Liveso.xml:80]: Sorry, I meant in front of the names of the prophets: "On the Prophet Isaiah"

File: 5001-6000/LIT5849GenealogyApostles.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply 5001-6000/LIT5849GenealogyApostles.xml:1]: ok

**Comment by thea-m** [comment]
I think this can be merged, but have the references to the deleted works been updated in the concerned manuscript records?

**Review by thea-m — APPROVED** [review]

**Pull request merged**